src/main/components/Mockup.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useContext } from 'react';
+import NameForm from './NameForm.jsx';
+import MockupName from './MockupName.jsx';
+import DataTree from './DataTree.jsx';
+import Button from './Button.jsx';
+import AssertionsForm from './AssertionsForm.jsx';
+import { TestsContext } from '../testsContext';
+
+
```

Comment:
Once again, I think the dataTreeOptions can be set up within a reducer if we implement redux. This would make the code more modular and more readable.